domain/src/main/kotlin/no/nav/su/se/bakover/domain/oppdrag/avstemming/Avstemmingsnøkkel.kt

```diff
@@ -6,10 +6,19 @@ import no.nav.su.se.bakover.common.startOfDay
 import java.time.LocalDate
 import kotlin.math.pow
 
-object Avstemmingsnøkkel {
-    // Avoid Utils.now() because of truncation to millis.
-    fun generer(tidspunkt: Tidspunkt = Tidspunkt.now()) =
-        tidspunkt.instant.epochSecond * 10.0.pow(9).toLong() + tidspunkt.nano
+data class Avstemmingsnøkkel(
+    val tidspunkt: Tidspunkt = Tidspunkt.now()
+) {
+    val nøkkel = generer(tidspunkt)
```

Comment:
Samme med denne, kunne låses og bruke compare/delegate for sammenligning?

Reply:
Denne er nå låst og kun eksponert via toString()